Cargo.toml

```diff
@@ -25,7 +24,7 @@ version = "1"
 features = ["derive"]
 
 [features]
-serde = ["optional/serde", "dep:serde"]
```

Comment:
Is this `serde` feature flag redundant now? Since we already have the optional serde dependency, can we delete this?

Reply:
Yeah, makes sense. I'll remove it 👍🏼